server/src/main/java/org/apache/kafka/server/config/Defaults.java

```diff
@@ -175,7 +175,7 @@ public class Defaults {
 
     /** ********* Share coordinator configuration *********/
     public static final int SHARE_COORDINATOR_STATE_TOPIC_PARTITIONS = 50;
-    public static final short SHARE_COORDINATOR_STATE_TOPIC_REPLICATION_FACTOR = 3;
+    public static final short SHARE_COORDINATOR_STATE_TOPIC_REPLICATION_FACTOR = 1; // in case single broker
```

Comment:
The way this is supposed to work is that the default is 3, and the single-broker config files set it to 1. That should be the final state when we are finished, so doing it this way for now is fine.